custom.css

```diff
@@ -1,3 +1,15 @@
+/* Center image via hashtag at end of img src */
+img[src*='#center'] {
```

Comment:
Wow,  I don't know about this trick ! Can you share some reference for further reading ?